src/lib/fast_check.py

```diff
@@ -160,7 +157,7 @@ def __init__(self, gcp_session: ClientSession, dt_session: ClientSession, token:
         self.logging_context = logging_context
         self.token = token
 
-    async def is_project_ready_to_monitor(self, project_id):
+    async def is_project_ready_to_monitor(self, project_id: str) -> Tuple[str, bool]:
```

Comment:
I am not sure why we are returning parameter, I mean you get project_id and return project_id??? This signature seems weird.

Reply:
This is likely due to how we are iterating the returning params later, we need the relation project_id - is_project_ready. And it's a class method (MetricsFastCheck), not a general one, but if you are strongly against it, I can try and refactor it as well.